### DISCUSSION TITLE
Add React Native/Expo support in docs

### DISCUSSION THREAD

**Discussion by firasrg** [mswjs/msw#1412] [Ideas]
Hello! I'm mobile developer, I use [Expo](https://expo.dev/) to build mobile apps.

 I'd like to thank you for the `msw/native` which help us mock APIs in our apps! Now we'd like to see it in [your docs](https://mswjs.io/docs/). So it becomes more appreciate by community, users and visitors.

I have already, built a cool example and saved it on [a Github repo called "expo-msw"](https://github.com/firas1220/expo-msw), as an initiative. 

So what do you think? I can even do that by myself then submit a PR.

Just want to know what is your answer?

thanks

**Comment by MattAgn** [comment]
That would be a great addition! Especially since Expo is becoming more and more the go-to to build RN apps

**Comment by gggfox** [comment]
This would be an awesome addition, specially with expo-router, msw is intercepting my call but its not mapping them to the defined handlers.
